v3.0/reference/transactions/transaction-pessimistic.md

```diff
@@ -30,56 +26,39 @@ TiDB 默认使用乐观事务模式，存在事务提交时因为冲突而失败
 
 - 乐观事务和悲观事务可以共存，事务可以任意指定使用乐观模式或悲观模式来执行。
 
-## 悲观事务的启用方法
-
-因为悲观事务目前还是一个试验性特性，默认是关闭的，启用悲观事务首先需要在 TiDB 的配置文件里添加：
+## 悲观事务的使用方法
 
-```
-[pessimistic-txn]
-enable = true
-```
-
-`enable` 设置为 `true` 以后，默认的事务模式仍然是乐观事务模式，要进入悲观事务模式有以下三种方式：
+进入悲观事务模式有以下三种方式:
 
 - 执行 `BEGIN PESSIMISTIC;` 语句开启的事务，会进入悲观事务模式。
 可以通过写成注释的形式 `BEGIN /*!90000 PESSIMISTIC */;` 来兼容 MySQL 语法。
 
 - 执行 `set @@tidb_txn_mode = 'pessimistic';`，使这个 session 执行的所有显式事务（即非 autocommit 的事务）都会进入悲观事务模式。
 
-- 在配置文件设置中默认启用悲观事务模式，使除了自动提交的单语句事务之外的所有事务都会进入悲观事务模式。
-
-    ```
-    [pessimistic-txn]
-    enable = true
-    default = true
-    ```
+- 执行 `set @@global.tidb_txn_mode = 'pessimistic';`，使之后整个集群所有新创建的 session 都会进入悲观事务模式执行显示事务。
```

Comment:
```suggestion
- 执行 `set @@global.tidb_txn_mode = 'pessimistic';`，使之后整个集群所有新创建的 session 都会进入悲观事务模式执行显式事务。
```

Reply:
This also needs to be fixed for the `dev` version.

Reply:
@juliezhang1112 Please update as suggested above. Thanks.